exporter.go

```diff
@@ -14,12 +14,20 @@ import (
 
 	"github.com/buildpacks/lifecycle/api"
 	"github.com/buildpacks/lifecycle/buildpack"
-	"github.com/buildpacks/lifecycle/cmd"
 	"github.com/buildpacks/lifecycle/launch"
 	"github.com/buildpacks/lifecycle/layers"
 	"github.com/buildpacks/lifecycle/platform"
 )
 
+const (
+	EnvLayersDir         = "CNB_LAYERS_DIR"
+	EnvAppDir            = "CNB_APP_DIR"
+	EnvPlatformAPI       = "CNB_PLATFORM_API"
+	EnvDeprecationMode   = "CNB_DEPRECATION_MODE"
+	EnvProcessType       = "CNB_PROCESS_TYPE"
+	DeprecationModeQuiet = "quiet"
```

Comment:
Why are these constants public?

Reply:
No reason except that I copied them from the `cmd` package, and they are public in that package even though they are only used in `cmd`.

Reply:
so maybe it's better to make them private?

Reply:
So actually after typing https://github.com/buildpacks/lifecycle/pull/721#discussion_r714285383 I realized that having these variables exported makes the `lifecycle` package more useful as a library that others might import. The `CNB_` variables are part of the spec so it makes sense to have them as part of the package interface.

Reply:
I don't have a strong opinion about it, but generally, as long as we don't see a specific usage for it right now, I think it's better to make them private. We can always change them to be public if needed in the future.

Reply:
Actually on removing the entanglement between the `cmd` and `platform` packages, these constants no longer need to be copied! Please see caaa415